awsutil/generate_credentials_test.go

```diff
@@ -179,24 +177,35 @@ func TestRetrieveCreds(t *testing.T) {
 }
 
 func TestGenerateCredentialChain(t *testing.T) {
+	// Create a shared creds file with a default profile
+	dir := t.TempDir()
+	profileWithDefault := path.Join(dir, "profile_with_default")
+	f, err := os.Create(profileWithDefault)
+	require.NoError(t, err)
+	_, err = f.Write([]byte("[default]\nregion=us-east-2\n"))
+	require.NoError(t, err)
+	require.NoError(t, f.Close())
+
 	cases := []struct {
-		name        string
-		opts        []Option
-		expectedErr string
+		name              string
```

Comment:
Is it possible to add a new test case to validate the SharedConfigProfileNotExistError error check?